examples/miscellaneous/clipboard.py

```diff
@@ -11,7 +11,7 @@
     orientation: 'vertical'
     BoxLayout:
         size_hint_y: None
-        height: self.minimum_height
+        height: self.height
```

Comment:
The property references itself and the value will just stay `100`. If compatibility with older versions is needed, GridLayout could be used.